src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -5,6 +5,7 @@
     <filter root="/apps/sling" />
 #end
 #if ( $isSingleCountryWebsite  == "n" )
-    <filter root="/apps/msm/__appsFolderName___blueprint" mode="merge"/>
+    <filter root="/apps/msm" mode="merge"/>

```

Comment:
@DominikSuess  Does "mode=merge"  make sense in the Immutable packages? Its unclear to me what this is merging with... from vendor packages? from potential other ui.apps?  ... it seems like if this could install into a space clearly owned by this package, it would be the best/more clear option? Im not sure if thats possible?

Reply:
@davidjgonzalez  - yes it makes  sense for the local dev experience. This was discussed multiple times the last days. It only does impact the local dev experience where the sling installer is in the game and packages might be installed at any time - but for those it is crucial to not accidentially wipe out content. For skyline, it may "hide" away some crimes in missing package dependencies (unclear ownership of structures) but it is a viable alteranative to what I think would be clearer to resolve via repoinit statements (create path) - yet we don't have clarity if we would want to support/recommend repoinit within projects.

Reply:
Adding the repo structure package allows to define there `/apps/msm` (https://github.com/adobe/aem-project-archetype/pull/256/commits/b4b94d13dfd5506e5e2d5d617b41fa3de66fb10a#diff-1e7b5b8139394ac52b9aa8cf3621c349R43), we can restrict here to our app's blueprints.